### ISSUE TITLE
Document global use of border:box

### ISSUE THREAD

**Issue by mattgreenfield** [createdotnet/wdk#39]
Some of the CSS we provide for widgets and shop pages (and maybe other places) require global border box. A lot of these CSS file also include it feature by feature, but it would be nice to remove that once legacy templates are not supported.

All the responsive templates have it set globally using `body *`,

Should we be documenting this for wdk users?

**Comment by benjaminparry** [comment]
Yes, I am looking at adding these as part of the boilerplate v2 as raised in this issue - https://github.com/createdotnet/wdk/issues/42 - please feel free to add some more suggestions.

**Comment by benjaminparry** [comment]
How time flies…
